### DISCUSSION TITLE
How to install rjava on archlinux

### DISCUSSION THREAD

**Discussion by ambarja** [s-u/rJava#299] [Q&A] [answered]
I am a new user in archlinux and I would like install rjava, but I have problems, someone could you help me ? 
Thanks

**Comment by s-u** [comment, marked as answer]
Maybe, if you include full details. Make sure you install Java JDK and configure R with Java support - those are minimal prerequisites. See also the Troubleshooting section of the [README](https://github.com/s-u/rJava/blob/master/README.md). If you run into issues, post the full output of the R configuration step (`javareconf`) and the `rJava` installation.

**Reply by ambarja** [reply]
Thank a lot @s-u, my solution was install the next dependeces on my distro :point_down: 

```
yay -S jdk-openjdk jre-openjdk jdk-openjdk-headless jre-openjdk-headless openjdk-src openjdk-doc
```
